client/components/layout/templates/sidebar.jade

```diff
@@ -2,13 +2,23 @@
   nav#layout-sidebar
     a#layout-sidebar-home( href='/' )
       include ../public/icons/logo.svg
-    a#layout-sidebar-new-article(
-      href='/articles/new'
-      class=(sd.URL == '/articles/new' ? 'is-active' : '')
-    )
-      include ../public/icons/layout_new_article.svg
-      br
-      | New Article
+    if user.isEditorial()
```

Comment:
This works since we're looking at the user's `current_channel` which is always accurate, but it might make more sense if we used [this method](https://github.com/artsy/positron/blob/master/client/models/channel.coffee#L15) on the channel instead. However I don't _think_ channel gets passed into locals atm so this template wouldn't have access to it yet unless each app passed that. Soo down the line it probably makes more sense to pass channel into https://github.com/artsy/positron/blob/master/client/lib/middleware.coffee#L14 but I think this is a good in-between. Not a blocker.